### PR TITLE
199 implement pro profile page

### PR DESCRIPTION
**Pull request by Lindt19**
To address #199 I implemented the professional profile page where professional users can update their profiles by adding their status, their domain of expertise, and the experience they have. This data can be used later in the forum, by displaying the status of the person answering a post for example.

### CODE REVIEW

File: app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+     */
+    fun checkAndLoadData(view: View){
+        if (!checkPolicy()) {
+            createSnackBar(view, getString(R.string.privacy_policy_not_accepted))
+        } else {
+            loadData()
+            createSnackBar(view, getString(R.string.changes_saved))
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (Lindt19) [reply app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt:120]: Good point 👍🏻

File: app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                showPolicy()
+            }
+
+            override fun updateDrawState(ds: TextPaint) {
+                super.updateDrawState(ds)
+                ds.isUnderlineText = true
+            }
+        }
+        val linkText = SpannableString(getString(R.string.privacy_policy))
+        linkText.setSpan(clickableSpan, 0, linkText.length, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
```
Comment (wiam8) [inline app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt:53]: You could import Spanned

File: app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        val cs = TextUtils.expandTemplate("I accept the ^1", linkText)
```
Comment (wiam8) [inline app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt:54]: Probably better as a string resource ;)

File: app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    private fun loadData() {
+        proStatus = findViewById<EditText>(R.id.proProfileStatusEditTxt).text.toString()
+        proDomain = findViewById<EditText>(R.id.proProfileDomainEditTxt).text.toString()
+        proExperience = findViewById<EditText>(R.id.proProfileExperienceEditTxt).text.toString()
+
+        val updatedProfile = ProUser(
+            id = VerificationActivity.currentUserId,
+            name = VerificationActivity.currentUserName,
+            proofName = VerificationActivity.currentUserProofName,
+            proofUri = VerificationActivity.currentUserProofUri,
```
Comment (wiam8) [inline app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProProfileActivity.kt:80]: Again, probably clearer to just import VerificationActivity

**Review by wiam8 — APPROVED** [review]
Always been a fan of your UI designs, great job!

**Pull request merged**